### DISCUSSION TITLE
Query on usage of Robot Framework with Pywinauto

### DISCUSSION THREAD

**Discussion by shailygajjar** [pywinauto/pywinauto#1220] [Q&A]
Can I use robot framework with pywinauto to automate my windows application?

So far, I have installed pywinauto, robotframework, wxpython, robotremoteserver. Is there any other library required to start with 
test automation?

**Comment by vasily-v-ryabov** [comment]
It should be easy to define your own Python steps in Robot Framework. I haven't personally used it but some users told me it was really easy.
